src/tasks/__tests__/processBulkUpload.int.test.ts

```diff
@@ -335,7 +371,8 @@ describe('processBulkUpload', () => {
 	});
 
 	it('should download, process, and resolve the bulk upload if the sourceKey is accessible and contains a valid CSV bulk upload', async () => {
-		await createTestBaseFields();
+		const [proposalSubmitterEmailFBaseField, organizationNameBaseField] =
```

Comment:
Typo in the variable name.

```suggestion
		const [proposalSubmitterEmailBaseField, organizationNameBaseField] =
```